.circleci/config.yml

```diff
@@ -0,0 +1,145 @@
+version: 2
+jobs:
+  test_npm:
+    docker:
+      - image: node:8.9.4
+    steps:
+      - checkout
+
+      - run:
+          name: Install dependencies
+          command: |
+            apt-get update
+            apt-get install -y shellcheck
+
+      - run:
+          name: Test npm install and ensure scripts are added to path
+          command: |
+            npm install -g git://github.com/reactiveops/rok8s-scripts.git#${CIRCLE_SHA1}
+
+      - run:
+          name: Run Shellcheck
+          command: |
+            shellcheck bin/docker-* \
+              bin/install-rok8s-requirements \
+              bin/k8s-apply \
+              bin/verify-deployment \
+              bin/k8s-lint \
+              bin/prepare-* \
+              bin/helm-*
+
+      - run:
+          name: Run basic linting
+          command: |
+            cd examples/production-ready
+            k8s-lint -f staging.config
+            k8s-lint -f production.config
+
+
+  test_pip:
+    environment:
+      PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/google-cloud-sdk/bin
+
+    docker:
+      - image: python:2.7
+    steps:
+      - checkout
+
+      - run:
+          name: Install dependencies
+          command: |
+            apt-get update
+            apt-get install -y shellcheck
+
+      - run:
+          name: Test pip install and ensure scripts are added to path
+          command: |
+            pip install git+https://github.com/reactiveops/rok8s-scripts.git@$CIRCLE_SHA1
```

Comment:
Would it be better to do `pip install .` so we are testing the installation of the already pulled files?

Reply:
Somehow I didn't realize that was an option - good idea!